constants.js

```diff
@@ -86,7 +86,8 @@ const checksWhitelist = {
       mergeConflictCheck,
       jobCheck,
       modelCheck,
-      codeOwnerCheck
+      codeOwnerCheck,
+      oldBuildLabelCheck
```

Comment:
Remove this from `[pushEvent]` and maybe also add `forcePushCheck` here? The second part can be done in different PR.

Reply:
Done.